executor/show_stats.go

```diff
@@ -156,3 +156,21 @@ func (e *ShowExec) valueToString(value *types.Datum, size int) (string, error) {
 	}
 	return str, nil
 }
+
+func (e *ShowExec) fetchShowStatsHealthy() {
+	do := domain.GetDomain(e.ctx)
+	h := do.StatsHandle()
+	dbs := do.InfoSchema().AllSchemas()
+	for _, db := range dbs {
+		for _, tbl := range db.Tables {
+			statsTbl := h.GetTableStats(tbl.ID)
+			if !statsTbl.Pseudo {
+				e.appendRow([]interface{}{
+					db.Name.O,
+					tbl.Name.O,
+					int64((1 - float64(statsTbl.ModifyCount)/float64(statsTbl.Count)) * 100.0),
```

Comment:
I think the healthy should never be negative.

How about
```
(statsTabl.Count / (statsTbl.ModifyCount + statsTbl.Count)) * 100.0

```

Reply:
Can we make sure that table count is not zero?

Reply:
How about there is only 2 row in the table and the modify-cout is also 2? Will the health value be 0?

Reply:
If use `(statsTabl.Count / (statsTbl.ModifyCount + statsTbl.Count))`, the score is always too high. Say we havenot analyze but the score will be 50.
I think if modifycount is equal or larger than totol count, the score will be 0. That is better.